sktm/patchwork.py

```diff
@@ -700,7 +706,8 @@ def get_patch_by_id(self, pid):
             set of supported attributes depends on which API versions are
             supported by a specific Patchwork instance.
         """
-        response = requests.get("%s/%d" % (self.apiurls.get("patches"), pid))
+        response = requests.get(join_with_slash(self.apiurls.get("patches"),
```

Comment:
This doesn't work (and similarly with some other calls below): `AttributeError: 'int' object has no attribute 'lstrip'`.